### ISSUE TITLE
Version 0.4.3 orderbook isn't kept updated

### ISSUE THREAD

**Issue by kenfehling** [coinbase/gdax-node#62]
I've created two test repos, one with 0.4.2 and one with 0.4.3 (master). They each print out the top bid and ask every few seconds.

[0.4.2 repo](https://github.com/kenfehling/gdax-orderbook-test-0.4.2)
[0.4.3 repo](https://github.com/kenfehling/gdax-orderbook-test-0.4.3)

I recorded results for each started at approximately the same time, not exactly so the values are slightly different, but the main finding is that after about a minute the 0.4.3 version begins to get stale on one side of the order book (in this instance the ask side, not sure if that's always the case).

0.4.2
```
2963.87 2962.96
2963.91 2962.94
2963.94 2963.86
2963.94 2963.28
2963.86 2959.02
2964.97 2961.73
2964.98 2964.21
2964.96 2964.2
2964.94 2964.16
2965.6 2964.61
```

0.4.3
```
2963.9 2962.51
2963.88 2962.51
2963.9 2963.79
2963.9 2963.79
2963.88 2962.51
2963.9 2962.51
2963.9 2964.21
2963.9 2964.2
2963.9 2964.16
2963.9 2964.55
```

**Comment by kenfehling** [comment]
Now I'm running my tests again and it's fine. Sorry if this is an issue on my end. I'll keep researching.

**Comment by kenfehling** [comment]
Actually it seems like the issue might happen when I use the new multiple product feature. I've updated the repo. Also it looks like it's not completely stale, just a lot slower to update, and it happens on both sides of the order book.

**Comment by awitherow** [comment]
Can you please let us know what node version and npm version you are using?

@fb55 can we do some sort of minimum issue template where creators have to input such rudimentary information?

**Comment by kenfehling** [comment]
node: v6.10.2
npm: 3.10.10

**Comment by awitherow** [comment]
Thanks @kenfehling the node/npm version is maybe for future users or for further debugging info.

**Comment by Rampage89** [comment]
I have experienced the same issue as well, one side of the orderbook is not kept up to date, mostly the asks..

**Comment by ambethia** [comment]
Do we think this is an issue with the `gdax-node` client itself or are the messages coming over the websockets connection delayed?

**Comment by kenfehling** [comment]
My guess is whatever the issue is, it started at some point after commit 98d044efba8329e0fd4f583ae2efaaf60ac81d8c on April 3 because it appears to happen on 0.4.3 but not 0.4.2

**Comment by mhuggins** [comment]
I can't even get the initial order book request to complete.  Perhaps a different issue, but I'm on 0.4.3.

**Comment by mhuggins** [comment]
I'm seeing a similar issue with both versions.  0.4.3 quickly gets out of sync with what appears on the live site, while 0.4.2 appears to stay in sync.

**Comment by 26medias** [comment]
This is not fixed, see the chart on #80

**Comment by rmm5t** [comment]
Closing in favor of #80.
